src/plugins/xfconf/README.md

```diff
@@ -0,0 +1,51 @@
+- infos = Information about the xfconf plugin is in keys below
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/xfconf
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained libc configurable experimental unfinished
+- infos/metadata =
+- infos/description = storage plugin for xfconf
+
+## Introduction
+
+This is a storage plugin to mount the xfconf configuration settings.
+
+## Dependencies
+
+xfconf from the XFCE project
+
+## Usage
+
+The usage is identical to most storage plugins except that the channel option during mount must be defined in order to tell xfconf which channel to mount.
+
+## Examples
+
+```sh
+# Backup-and-Restore: user:/tests/xfconf
+
+# mount the xfwm channel
+kdb mount /dev/null /test/xfwm xfconf "channel=xfwm4"
```

Comment:
This plugin also needs new-backend and new-style mounting. Let us discuss this later in the meeting.